### PR TITLE
Translate pytorch_vision_vgg.md

### PR DESCRIPTION
**Pull request by Jo-ad-fontes**
> ## 라이선스 동의
> _변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._ _더 자세한 내용은 [기여하기 문서](https://github.com/9bow/PyTorch-hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._ _동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._
> 
> * [x]  기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.
> 
> ## 관련 이슈 번호
> _이 Pull Request와 관련있는 이슈 번호를 적어주세요._ _이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_
> 
> * **이슈 번호**: [VGG-NETS 번역예정 번역 예정 #7](https://github.com/9bow/PyTorch-hub-kr/issues/7)
> 
> ## PR 종류
> _이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._
> 
> * [ ]  오탈자를 수정하거나 번역을 개선하는 기여
> * [x]  번역되지 않은 허브를 번역하는 기여
> * [ ]  공식 허브 내용을 반영하는 기여
> * [ ]  위 종류에 포함되지 않는 기여
> 
> ## PR 설명
> _이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
> 
> * 변경 사항
>   
>   * 파이토치 허브 모델 중 Vision 파트의 VGG-NETS을 번역하였습니다.
>   * 즉, pytorch_vision_vgg.md을 번역하였습니다.

### CODE REVIEW

File: pytorch_vision_vgg.md [hunk 1]
```diff
@@ -30,23 +30,22 @@ model = torch.hub.load('pytorch/vision:v0.9.0', 'vgg11', pretrained=True)
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+모든 사전 훈련된 모델은 훈련때와 같은 방식으로 정규화된 입력 이미지를 주어야합니다.
+즉, `(3 x H x W)` 모양의 3채널 RGB 이미지의 미니배치에서 `H`와 `W`는 최소 `224`가 될 것으로 예상됩니다.
+이미지는 `[0, 1]` 범위로 로드한 다음 `mean = [0.485, 0.456, 0.406]`과 `std = [0.229, 0.224, 0.225]` 값을 사용하여 정규화해야 합니다.
```
Comment (miNept) [inline pytorch_vision_vgg.md:35]: 이미지를 `[0,1]`범위로 로드한다는 것의 의미가 설명이 된다면 좋을 것 같아요!

Comment (Jo-ad-fontes) [reply pytorch_vision_vgg.md:35]: 좋습니다!

**Review by hyoyoung — APPROVED** [review]
looks good to me
